source/NHibernate.AspNet.Web.Specs/AssemblyStartup.cs

```diff
@@ -23,7 +23,7 @@ private void HostStart()
             var config = new SpecsForMvcConfig();
             config.UseIISExpress()
                 .With(Project.Named("NHibernate.AspNet.Web"))
-                //.UseMSBuildExecutableAt(@"C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\MSBuild\15.0\bin\msbuild.exe")
+                .UseMSBuildExecutableAt(@"C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\MSBuild\15.0\bin\msbuild.exe")
```

Comment:
On NHibernate side, we now use [`dotnet`](https://github.com/nhibernate/nhibernate-core/blob/5.0.0/Tools/msbuild.cmd) instead of `msbuild`. It is in the system path, so no more tedious locating of it. We still have some artifacts requiring [`msbuild`](https://github.com/nhibernate/nhibernate-core/blob/5.0.0/Tools/actual_msbuild.cmd): the documentation. This one relies on `vswhere` utility to get the `msbuild` available on the system. And NHibernate  build scripts handle downloading [NuGet](https://github.com/nhibernate/nhibernate-core/blob/5.0.0/build-common/common-project.xml#L44), [vswhere](https://github.com/nhibernate/nhibernate-core/blob/5.0.0/build-common/common-project.xml#L52) (with [packages.config](https://github.com/nhibernate/nhibernate-core/blob/5.0.0/Tools/packages.config#L11)) and other tools as required.

Reply:
I'm fighthing with appveyor. Everything works locally but once in appveyor 3 tests using SpecsFor don't work. I'll try to migrate to dotnet to see if I get any further.

Reply:
I fixed a typo to check if the build was still working and it failed. All specsfor test fail with this error:
```
[00:02:30] 4.) Container.GetInstance(SpecsFor.Mvc.MvcWebApp)
[00:02:30]  ---> OpenQA.Selenium.WebDriverException: Failed to start up socket within 45000 ms. Attempted to connect to the following addresses: 127.0.0.1:7055
[00:02:30]    at OpenQA.Selenium.Firefox.FirefoxDriverServer.ConnectToBrowser(TimeSpan timeToWait)
[00:02:30]    at OpenQA.Selenium.Firefox.FirefoxDriverServer.Start()
[00:02:30]    at OpenQA.Selenium.Firefox.FirefoxDriverCommandExecutor.Execute(Command commandToExecute)
[00:02:30]    at OpenQA.Selenium.Remote.RemoteWebDriver.Execute(String driverCommandToExecute, Dictionary`2 parameters)
[00:02:30]    at OpenQA.Selenium.Remote.RemoteWebDriver.StartSession(ICapabilities desiredCapabilities)
[00:02:30]    at OpenQA.Selenium.Remote.RemoteWebDriver..ctor(ICommandExecutor commandExecutor, ICapabilities desiredCapabilities)
[00:02:30]    at OpenQA.Selenium.Firefox.FirefoxDriver..ctor(ICapabilities capabilities)
[00:02:30]    at SpecsFor.Mvc.BrowserDriver.<.cctor>b__2()
[00:02:30]    at SpecsFor.Mvc.BrowserDriver.GetDriver()
[00:02:30]    at SpecsFor.Mvc.MvcWebApp..ctor()
[00:02:30]    at lambda_method(Closure , IBuildSession , IContext )
[00:02:30]    --- End of inner exception stack trace ---
[00:02:30]    at lambda_method(Closure , IBuildSession , IContext )
[00:02:30]    at StructureMap.Building.BuildPlan.Build(IBuildSession session, IContext context) in c:\BuildAgent\work\996e173a8ceccdca\src\StructureMap\Building\BuildPlan.cs:line 151
[00:02:30]    at StructureMap.BuildSession.BuildNewInSession(Type pluginType, Instance instance) in c:\BuildAgent\work\996e173a8ceccdca\src\StructureMap\BuildSession.cs:line 114
[00:02:30]    at StructureMap.Pipeline.NulloTransientCache.Get(Type pluginType, Instance instance, IBuildSession session) in c:\BuildAgent\work\996e173a8ceccdca\src\StructureMap\Pipeline\NulloTransientCache.cs:line 28
[00:02:30]    at StructureMap.BuildSession.ResolveFromLifecycle(Type pluginType, Instance instance) in c:\BuildAgent\work\996e173a8ceccdca\src\StructureMap\BuildSession.cs:line 103
[00:02:30]    at StructureMap.SessionCache.GetObject(Type pluginType, Instance instance, ILifecycle lifecycle) in c:\BuildAgent\work\996e173a8ceccdca\src\StructureMap\SessionCache.cs:line 93
[00:02:30]    at StructureMap.SessionCache.GetDefault(Type pluginType, IPipelineGraph pipelineGraph) in c:\BuildAgent\work\996e173a8ceccdca\src\StructureMap\SessionCache.cs:line 68
[00:02:30]    at StructureMap.Container.GetInstance(Type pluginType) in c:\BuildAgent\work\996e173a8ceccdca\src\StructureMap\Container.cs:line 336
[00:02:30]    at StructureMap.Container.GetInstance[T]() in c:\BuildAgent\work\996e173a8ceccdca\src\StructureMap\Container.cs:line 201
[00:02:30]    at StructureMap.AutoMocking.AutoMocker`1.get_ClassUnderTest() in c:\BuildAgent\work\996e173a8ceccdca\src\StructureMap\AutoMocking\AutoMocker.cs:line 128
[00:02:30]    at SpecsFor.SpecsForEngine`1.InitializeClassUnderTest()
[00:02:30]    at SpecsFor.SpecsFor`1.InitializeClassUnderTest()
[00:02:30]    at SpecsFor.SpecsFor`1.SpecsFor.ISpecs.InitializeClassUnderTest()
[00:02:30]    at SpecsFor.SpecsForEngine`1.Init()
[00:02:30] 
[00:02:30]    at SpecsFor.SpecsForEngine`1.Init()
[00:02:30]    at SpecsFor.SpecsFor`1.SetupEachSpec()
```